### PR TITLE
Revert "[glib]Fix missing dependency selinux."

### PR DESCRIPTION
**Pull request by grdowns**
Reverts microsoft/vcpkg#7988

### CODE REVIEW

File: ports/glib/CONTROL [hunk 1]
```diff
@@ -3,6 +3,3 @@ Version: 2.52.3-14-3
 Homepage: https://developer.gnome.org/glib/
 Description: Portable, general-purpose utility library.
 Build-Depends: zlib, pcre, libffi, gettext, libiconv
```
Comment (cbezault) [inline ports/glib/CONTROL]: Make selinux a default feature.

**Review by cbezault — CHANGES_REQUESTED** [review]

File: ports/glib/CONTROL (hunk 1 above)
Comment (JackBoosY) [reply ports/glib/CONTROL]: @cbezault Selinux may not be installed by default in linux, do you insist on this?

Comment (cbezault) [reply ports/glib/CONTROL]: Yes, it should be default, selinux is pretty important, it's just that glib should be buildable without it.

Comment (grdowns) [reply ports/glib/CONTROL]: I dug into this a bit more. Selinux is not required for all builds of glib (we know this), caused my machine to be unbootable, and was uninstallable through a Docker image. We should leave this as a non-default feature. Thanks @JackBoosY!

**Pull request closed without merging**